src/types.ts

```diff
@@ -54,7 +54,7 @@ export type UpdateFunctionFile =
 
 export type UpdateFunction = (
   file: UpdateFunctionFile
-) => string | File | null | Promise<string | File | null>;
+) => string | File | null | Symbol | Promise<string | File | null | Symbol>;
```

Comment:
If the update function returns `null` it means no file changes.
If the update function return a `symbol` it triggers a file deletion.

Reply:
Is there a way to type the Symbol to ensure it is the actual `deleteFile` symbol?

Reply:
I don't know, I actually never worked with types for Symbols